app/views/spree/shared/_header.html.erb

```diff
@@ -1,15 +1,29 @@
-<div id="spree-header">
+<!-- <div id="spree-header">
   <header id="header" data-hook>
     <div class="container">
       <div class="row">
         <figure id="logo" class="col-md-4 col-sm-3" data-hook>
           <%= logo %>
         </figure>
-        <%= render partial: 'spree/shared/nav_bar' %>
+        <%#= render partial: 'spree/shared/nav_bar' %>
       </div>
     </div>
   </header>
   <div class="container">
-    <%= render partial: 'spree/shared/main_nav_bar' %>
+    <%#= render partial: 'spree/shared/main_nav_bar' %>
   </div>
-</div>
+</div>  -->
+
+<header id="header" data-hook>
```

Comment:
This spree default id: "spree-header" is missing. Add this id to div under nav tag